src/ip_defer.c

```diff
@@ -73,8 +73,10 @@ void ip_defer_handler(int delta_time __unused)
     defer_inhibit = 1;
     while (1) {
         struct ip_defer *ipd = ip_defer_peek();
-        if (!ipd)
+        if (!ipd) {
+            defer_inhibit = 0;
```

Comment:
Let's change the type of `defer_inhibit` from unsigned int to `bool` since it is intended to operate binary.

Reply:
Done.